basic_bootloader/src/bootloader/errors.rs

```diff
@@ -109,6 +109,15 @@ pub enum InvalidTransaction {
     /// Protocol upgrade txs should always be successful.
     // TODO: it's not really a validation error
     UpgradeTxFailed,
+
+    /// Transaction makes the block reach the gas limit
```

Comment:
Initially this enum was supposed to be about tx validation, right?

Reply:
Ok, this is just that it doesn't come from validation step - we need to execute tx first. But this is ok imo, just a note